js/src/dapps/registry/ui/address.js

```diff
@@ -58,41 +75,46 @@ class Address extends Component {
           </abbr>
         </a>
       );
-    } else {
-      caption = (
-        <code className={ styles.align }>
-          { shortenHash ? (
-            <Hash
-              hash={ address }
-              linked
-            />
-          ) : address }
-        </code>
-      );
     }
 
     return (
-      <div
-        key={ key }
-        className={ styles.container }
-      >
-        <IdentityIcon
-          address={ address }
-          className={ styles.align }
-        />
-        { caption }
-      </div>
+      <code className={ styles.align }>
+        { shortenHash ? (
+          <Hash
+            hash={ address }
+            linked
+          />
+        ) : address }
+      </code>
     );
   }
 }
 
+function mapStateToProps (initState, initProps) {
+  const { accounts, contacts } = initState;
+
+  const allAccounts = Object.assign({}, accounts.all, contacts);
+
+  // Add lower case addresses to map
```

Comment:
It's just because sometimes the returned data from a contract method isn't formatted as an address. This does not change the displayed address, it only adds to the Address Map the key for the lower case addresses.